src/mapping/tests/RadialBasisFctMappingTest.cpp

```diff
@@ -55,77 +55,54 @@ ReferenceSpecification format:
 - -1 on rank means all ranks
 - v is the expected value of n-th vertex on that particular rank
 */
-using MeshSpecification = std::vector<VertexSpecification>;
+struct MeshSpecification {
+  std::vector<VertexSpecification> vertices;
+  int                              meshDimension;
+  mesh::PtrMesh                    meshName;
```

Comment:
This should be the name of the mesh, not the mesh itself

Reply:
Do you mean instead of having `mesh::PtrMesh meshName` as a member, it should have `std::string meshName` as a member?